Sources/CorePayments/Networking/Enums/Environment.swift

```diff
@@ -8,9 +8,9 @@ public enum Environment {
     var baseURL: URL {
         switch self {
         case .sandbox:
-            return URL(string: "https://api.sandbox.paypal.com")!
+            return URL(string: "https://api-m.sandbox.paypal.com")!
         case .live:
-            return URL(string: "https://api.paypal.com")!
+            return URL(string: "https://api-m.paypal.com")!
         }
```

Comment:
I c, so it's more efficient to use this in our network calls overall?

Reply:
Do we need to check all features with new url?

Reply:
Seems to work fine on all demo features.

Reply:
> I c, so it's more efficient to use this in our network calls overall?

Yeah it seems like it will be because of some CDN network edge optimization.